api/src/main/java/com/alibaba/nacos/api/utils/ClassUtils.java

```diff
@@ -0,0 +1,139 @@
+package com.alibaba.nacos.api.utils;
+
+import java.io.File;
+import java.lang.reflect.Modifier;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.List;
+import java.util.jar.JarFile;
+import java.net.JarURLConnection;
+import java.util.jar.JarEntry;
+
+/**
+ * Get subclass.
+ *
+ * @author dingjuntao
+ * @date 2021/7/8 19:26
+ */
+public class ClassUtils {
```

Comment:
Should we move this utils to common module. 
If the utils in api module means can provide users to use, but I think this class is an inner utils.

BTW, the jar scan has high risk and depend env strongly. For example, there is much difference between running in IDEA and running by package. Did you has enough test for this?

Reply:
I have tested it, and it can run normally in the jar package.
In addition, I have removed ClassUtils.java and put all functions in PayLoaderProvider.java .